README: suggest faucet for testnets (#338)
ava-labs/teleporter: README.md

./scripts/testnet/run_testnet_e2e_flows.sh
```

> The user wallet set in `.env` must have native tokens for each of the subnets used in order for the test flows to be able to send transactions on those networks, which can be obtained for public testnet subnets at [the Avalanche Testnet Faucet](https://core.app/tools/testnet-faucet/?subnet=dispatch&token=dispatch).

Reviewer: ```suggestion
The user wallet set in `.env` must have native tokens for each of the subnets used in order for the test flows to be able to send transactions on those networks, which can be obtained for public testnet subnets at [the Avalanche Testnet Faucet](https://core.app/tools/testnet-faucet).
```

Let's not link to a specific chain/asset

> The user wallet set in `.env` must have native tokens for each of the subnets used in order for the test flows to be able to send transactions on those networks, which can be obtained for public testnet subnets at [the Avalanche Testnet Faucet](https://core.app/tools/testnet-faucet/?subnet=dispatch&token=dispatch).

Reviewer 2: ```suggestion
The user wallet set in `.env` must have native tokens for each of the Subnets used in order for the test flows to be able to send transactions on those networks. The [Avalanche Testnet Faucet](https://core.app/tools/testnet-faucet) can be used to obtain native tokens for certain public testnet Subnets.
```

Nits: Subnets should be capitalized, and not all testnet Subnets will necessarily be supported by the faucet.
